### ISSUE TITLE
Problem with IE?

### ISSUE THREAD

**Issue by Shadowstep33** [inorganik/countUp.js#93]
I have like IE 9 I think; sorry for not having specifics. CountUp has not worked for me on any version of IE I have tried. Is there some feature that is the main problem that I can check for? It'd be great to just have a if(!window.requestAnimationFrame) or something along those lines. 

CountUp is returning '---' in IE which is harming a critical part of the game I am working on.

**Comment by inorganik** [comment]
I just tested in IE8, IE9, and IE11. It's works in all 3. If you look at the script, there is already an `if (!window.requestAnimationFrame)`. Try some debugging - check if you are passing valid values for startVal and endVal, put some log statements and break points to see what's going on.

**Comment by Shadowstep33** [comment]
Well, it worked in Chrome and Firefox... Only place it doesn't work is IE. I am using bitwise operators and parseInt, so maybe those don't work in IE. Or maybe IE just can't parse numbers from .text() 

I saw in the code '--' is returned when a NaN is supplied/parsed. I was in IE11 btw. Maybe I have an old version of CountUp or something. I pulled it a few months ago.

But in case someone else sees this and agrees with me that there is a problem in IE, this seemed to be a reliable fix:

```
				var goldNumber = new CountUp("gold-number", (lastGold | (parseInt(data.character.Gold) | 0) ), parseInt(data.character.Gold), 0, 1, options);
				goldNumber.start(function(){
//Possibly check for $("#gold-number").text() == "--" if you want
					$("#gold-number").text(data.character.Gold).digits();
				});
```
